src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -0,0 +1,94 @@
+import Url from 'url-parse';
+import pipe from 'ramda/src/pipe';
+import getEnvironment from '#app/routes/utils/getEnvironment';
+import { getUrlPath } from '../../../lib/utilities/urlParser';
+import handleError from '../handleError';
+import { Services, Variants } from '../../../models/types/global';
+
+interface UrlConstructParams {
+  pathname: string;
+  pageType: 'article' | 'cpsAsset' | 'topic' | 'live';
```

Comment:
Can we do something like this: 
- Create a new const for page types (we might need a better name, because there is already a PageTypes object in ../../../models/types/global

```
const PAGE_TYPES = {
  ARTICLE: 'article',
  CPS_ASSET: 'cpsAsset',
  HOME: 'home',
  TOPIC: 'topic',
  LIVE: 'live'
} as const;
```

Then use that to declare our page types - just trying to avoid duplication and improve readability
```suggestion
  pageType: keyof typeof PAGE_TYPES
```

Reply:
Also, could we also include the home page type here - might need to update the homepage/getInitialData accordingly